src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
```

Comment:
Yea a global logger would be the way. 

Hmm, let me think about the `DEBUG` problem.  It makes sense for it to override.